.github/workflows/rocm_wheels.yml

```diff
@@ -0,0 +1,66 @@
+name: rocm-wheels
+
+on:
+  push:
+    branches:
+      - develop
+  workflow_dispatch:
```

Comment:
Can we trigger this on PRs that might change files that are built? Similar to here: https://github.com/facebookresearch/xformers/blob/46bc6b5c5b91b4a2da3de14d0aee4f24dfe238ec/.github/workflows/win-build.yml#L4-L11
Also I would rename this file (& workload) as `rocm_build` - even though it builds a wheel, it's distinct from the wheels build+upload CI system and could be confusing otherwise.